test-utils/src/lib.rs

```diff
@@ -158,4 +169,14 @@ mod tests {
         let result = split_value(&mut rng, value);
         assert_eq!(value, result.iter().sum());
     }
+
+    #[rstest]
+    #[trace]
+    #[case(Seed::from_entropy())]
+    fn random_string_test(#[case] seed: Seed) {
+        let mut rng = make_seedable_rng(seed);
+
+        let result = random_string(&mut rng, 1..100);
+        assert!(result.chars().all(|c| c.is_ascii_alphanumeric()));
```

Comment:
It's better to rename `random_string` to suggest that it can only produce alphanum characters